webui/src/pages/list/ListQuery.css

```diff
@@ -0,0 +1,17 @@
+header {
```

Comment:
all the styling is done with CSS-in-JS (see https://material-ui.com/styles/basics/), so this piece should be done the same way (that is, not in pure CSS).

Reply:
Oh I just forgot to do it. That´s simple! 😉 
May I do it when fixing #539 which is another simple issue?

Reply:
I did a commit right now. ;-)